_candidates_en/KaiSasaki_1.md

```diff
@@ -0,0 +1,24 @@
+---
+name: Kai Sasaki
+title: "Lessons learned while implementing my own distributed deep learning framework on Spark"
+length: 40
+audience: Intermediate
+language: English
```

Comment:
@yohei1126 
According to https://github.com/scalajp/2018.scalamatsuri.org/issues/129, The language looks incorrect.

Reply:
Thanks. Fixed it. https://github.com/scalajp/2018.scalamatsuri.org/pull/328